js_tests/admin/RelatedObjectLookups.test.js

```diff
@@ -0,0 +1,21 @@
+module("admin.RelatedObjectLookups");
+test("html_unescape", function(assert) {
+  function unescape(then, expected, message) {
```

Comment:
`unescape` is a built in function ([deprecated](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/unescape), though), I would advise against using the same name for different purpose.